v1.3.1 breaks custom post type categories

I have a custom post type registered in my wordpress, and since the v1.3.1 upgrade, I cannot query categories on this custom post type. I don't think it was a bug being fixed in this release.

The field simply  doesn't exist anymore.

 "gatsby-source-wordpress": "^5.1.0"
WP-Gatsby Version 1.0.6
WP-Graphql Version 1.3.1

I query this data from gatsby, and I checked in the WP graphql IDE if it was Gatsby or WP-graphql related. This really comes from the upgrade from wp-graphql 1.2.6 to 1.3.1

Here is how I register my custom post type:
```php
add_action('init', function () {
    register_post_type('shows', [
        'label' => __('Émission', 'aro_theme'),
        'description' => __('Émissions', 'aro_theme'),
        'labels' => [
            'name' => _x('Émissions', 'Post Type General Name', 'aro_theme'),
            'singular_name' => _x('Émission', 'Post Type Singular Name', 'aro_theme'),
            'menu_name' => __('Émissions', 'aro_theme'),
            'name_admin_bar' => __('Émissions', 'aro_theme'),
        ],
        'supports' => ['title', 'editor', 'author', 'comments', 'revisions', 'excerpt'],
        'taxonomies' => ['category'],
        'hierarchical' => false,
        'public' => true,
        'show_ui' => true,
        'show_in_menu' => true,
        'menu_position' => 5,
        'menu_icon' => 'dashicons-format-audio',
        'show_in_admin_bar' => true,
        'show_in_nav_menus' => true,
        'can_export' => true,
        'has_archive' => true,
        'exclude_from_search' => false,
        'publicly_queryable' => true,
        'capability_type' => 'page',

        'rewrite' => [
            'slug' => 'emissions',
            'with_front' => false,
        ],
        'cptp_permalink_structure' => '/%category%/%postname%/',

        'show_in_graphql' => true,
        'graphql_single_name' => 'show',
        'graphql_plural_name' => 'shows',
    ]);
}, 0);
```

Here is one of my graphql query:
```
  query($id: String!) {
    wpShow(id: { eq: $id }) {
      uri
      title
      content
      date
      categories {
        nodes {
          uri
          name
        }
      }
      author {
        node {
          uri
          name
        }
      }
      meta {
        file {
          mediaItemUrl
        }
      }
    }
  }
```

And some logs from Gatsby:

```
File: src/templates/show.js:63:7

failed extract queries from components - 0.084s
success write out requires - 0.001s

 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed


/home/jo/git/git.oolsa.net/ext/aro/website/src/templates/author.js
  75:9  error  Cannot query field "categories" on type "WpShow"  graphql/template-strings

/home/jo/git/git.oolsa.net/ext/aro/website/src/templates/show.js
  63:7  error  Cannot query field "categories" on type "WpShow"  graphql/template-strings

✖ 2 problems (2 errors, 0 warnings)


failed Re-building development bundle - 0.248s
ERROR in 
/home/jo/git/git.oolsa.net/ext/aro/website/src/templates/author.js
  75:9  error  Cannot query field "categories" on type "WpShow"  graphql/template-strings

/home/jo/git/git.oolsa.net/ext/aro/website/src/templates/show.js
  63:7  error  Cannot query field "categories" on type "WpShow"  graphql/template-strings

✖ 2 problems (2 errors, 0 warnings)


webpack compiled with 1 error

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,10 +2,41 @@
 
 ## Upcoming
 
+## 1.3.2
+
+### Bugfix
+
+- Fixes ([#1802](https://github.com/wp-graphql/wp-graphql/issues/1802)) by reversing a change to how initial post types and taxonomies are setup.
+
+## 1.3.1
+
+### Bugfix
+
+- patches a bug where default post types and taxonomies disappeared from the Schema
+
+## 1.3.0
+
+### Noteable changes
+
+Between this release and the prior release ([v1.2.6](https://github.com/wp-graphql/wp-graphql/releases/tag/v1.2.6)) includes changes to pagination under the hood.
+
+While these releases correcting mistakes and buggy behavior, it's possible that workarounds have already been implemented either in the server or in client applications.
+
+For example, there was a bug with `start/end` cursors being reversed for backward pagination.
+
+If a client application were reversing the cursors to fix the issue, the reversal in the client will now _cause_ the issue.
+
+It's recommended to test your applications against this release, _specifically_ in regards to pagination.
+
 ### Bugfixes / Chores
 
-- Resolve backward pagination bug when using the post__in argument for post connection queries.
-
+- ([#1797](https://github.com/wp-graphql/wp-graphql/pull/1797)): Update test environment to allow custom permalink structures to be better tested. Moves the "show_in_graphql" setup of core post types and taxonomies into the `register_post_type_args` and `register_taxonomy_args` filters instead of modifying global filters directly.
+- ([#1794](https://github.com/wp-graphql/wp-graphql/pull/1794)): Cleanup to PHPStan config. Thanks @szepeviktor!
+- ([#1795](https://github.com/wp-graphql/wp-graphql/pull/1795)) and ([#1793](https://github.com/wp-graphql/wp-graphql/pull/1793)): Don't throw errors when external urls are provided as input for queries that try and resolve by uri
+- ([#1792](https://github.com/wp-graphql/wp-graphql/pull/1792)): Add missing descriptions to various fields in the Schema. Thanks @markkelnar!
+- ([#1791](https://github.com/wp-graphql/wp-graphql/pull/1791)): Update where `WP_GRAPHQL_URL` is defined to follow recommendation from WordPress.org.
+- ([#1784](https://github.com/wp-graphql/wp-graphql/pull/1784)): Fix `UsersConnectionSearchColumnEnum` to show the proper values that were accidentally replaced.
+- ([#1781](https://github.com/wp-graphql/wp-graphql/pull/1781)): Fixes various bugs related to pagination. Between this release and the v1.2.6 release the following bugs have been worked on in regards to pagination: ([#1780](https://github.com/wp-graphql/wp-graphql/pull/1780), [#1411](https://github.com/wp-graphql/wp-graphql/pull/1411), [#1552](https://github.com/wp-graphql/wp-graphql/pull/1552), [#1714](https://github.com/wp-graphql/wp-graphql/pull/1714), [#1440](https://github.com/wp-graphql/wp-graphql/pull/1440))
 ## 1.2.6
 
 ### Bugfixes / Chores

--- a/readme.txt
+++ b/readme.txt
@@ -4,7 +4,7 @@ Tags: GraphQL, API, Gatsby, Headless, Decoupled, React, Nextjs, Vue, Apollo, RES
 Requires at least: 5.0
 Tested up to: 5.6
 Requires PHP: 7.1
-Stable tag: 1.3.1
+Stable tag: 1.3.2
 License: GPL-3
 License URI: https://www.gnu.org/licenses/gpl-3.0.html
 

--- a/src/WPGraphQL.php
+++ b/src/WPGraphQL.php
@@ -125,7 +125,7 @@ final class WPGraphQL {
 
 		// Plugin version.
 		if ( ! defined( 'WPGRAPHQL_VERSION' ) ) {
-			define( 'WPGRAPHQL_VERSION', '1.3.1' );
+			define( 'WPGRAPHQL_VERSION', '1.3.2' );
 		}
 
 		// Plugin Folder Path.

--- a/src/WPGraphQL.php
+++ b/src/WPGraphQL.php
@@ -89,8 +89,8 @@ final class WPGraphQL {
 	 * The whole idea of the singleton design pattern is that there is a single object
 	 * therefore, we don't want the object to be cloned.
 	 *
-	 * @return void
 	 * @since  0.0.1
+	 * @return void
 	 */
 	public function __clone() {
 
@@ -102,8 +102,8 @@ final class WPGraphQL {
 	/**
 	 * Disable unserializing of the class.
 	 *
-	 * @return void
 	 * @since  0.0.1
+	 * @return void
 	 */
 	public function __wakeup() {
 
@@ -115,8 +115,8 @@ final class WPGraphQL {
 	/**
 	 * Setup plugin constants.
 	 *
-	 * @return void
 	 * @since  0.0.1
+	 * @return void
 	 */
 	private function setup_constants() {
 
@@ -154,8 +154,8 @@ final class WPGraphQL {
 	 * Include required files.
 	 * Uses composer's autoload
 	 *
-	 * @return bool
 	 * @since  0.0.1
+	 * @return bool
 	 */
 	private function includes() {
 
@@ -182,7 +182,7 @@ final class WPGraphQL {
 
 				add_action(
 					'admin_notices',
-					function() {
+					function () {
 
 						if ( ! current_user_can( 'manage_options' ) ) {
 							return;
@@ -224,8 +224,7 @@ final class WPGraphQL {
 	}
 
 	/**
-	 * Sets up actions to run at certain spots throughout WordPress and the WPGraphQL execution
-	 * cycle
+	 * Sets up actions to run at certain spots throughout WordPress and the WPGraphQL execution cycle
 	 *
 	 * @return void
 	 */
@@ -318,8 +317,9 @@ final class WPGraphQL {
 	 * If the server is running a lower version than required, throw an exception and prevent
 	 * further execution.
 	 *
-	 * @return void
 	 * @throws Exception
+	 *
+	 * @return void
 	 */
 	public function min_php_version_check() {
 
@@ -389,15 +389,8 @@ final class WPGraphQL {
 		);
 
 		// Filter how metadata is retrieved during GraphQL requests
-		add_filter(
-			'get_post_metadata',
-			[
-				'\WPGraphQL\Utils\Preview',
-				'filter_post_meta_for_previews',
-			],
-			10,
-			4
-		);
+		add_filter( 'get_post_metadata', [ '\WPGraphQL\Utils\Preview', 'filter_post_meta_for_previews' ], 10, 4 );
+
 	}
 
 	/**
@@ -413,72 +406,54 @@ final class WPGraphQL {
 	/**
 	 * This sets up built-in post_types and taxonomies to show in the GraphQL Schema
 	 *
-	 * @return void
 	 * @since  0.0.2
+	 * @return void
 	 */
 	public static function show_in_graphql() {
 
-		add_filter(
-			'register_post_type_args',
-			function( $args, $post_type ) {
+		global $wp_post_types, $wp_taxonomies;
 
-				if ( 'attachment' === $post_type ) {
-					$args['show_in_graphql']     = true;
-					$args['graphql_single_name'] = 'mediaItem';
-					$args['graphql_plural_name'] = 'mediaItems';
-				}
+		// Adds GraphQL support for attachments.
+		if ( isset( $wp_post_types['attachment'] ) ) {
+			$wp_post_types['attachment']->show_in_graphql     = true;
+			$wp_post_types['attachment']->graphql_single_name = 'mediaItem';
+			$wp_post_types['attachment']->graphql_plural_name = 'mediaItems';
+		}
 
-				if ( 'page' === $post_type ) {
-					$args['show_in_graphql']     = true;
-					$args['graphql_single_name'] = 'page';
-					$args['graphql_plural_name'] = 'pages';
-				}
+		// Adds GraphQL support for pages.
+		if ( isset( $wp_post_types['page'] ) ) {
+			$wp_post_types['page']->show_in_graphql     = true;
+			$wp_post_types['page']->graphql_single_name = 'page';
+			$wp_post_types['page']->graphql_plural_name = 'pages';
+		}
 
-				if ( 'post' === $post_type ) {
-					$args['show_in_graphql']     = true;
-					$args['graphql_single_name'] = 'post';
-					$args['graphql_plural_name'] = 'posts';
-				}
+		// Adds GraphQL support for posts.
+		if ( isset( $wp_post_types['post'] ) ) {
+			$wp_post_types['post']->show_in_graphql     = true;
+			$wp_post_types['post']->graphql_single_name = 'post';
+			$wp_post_types['post']->graphql_plural_name = 'posts';
+		}
 
-				return $args;
+		// Adds GraphQL support for categories.
+		if ( isset( $wp_taxonomies['category'] ) ) {
+			$wp_taxonomies['category']->show_in_graphql     = true;
+			$wp_taxonomies['category']->graphql_single_name = 'category';
+			$wp_taxonomies['category']->graphql_plural_name = 'categories';
+		}
 
-			},
-			10,
-			2
-		);
+		// Adds GraphQL support for tags.
+		if ( isset( $wp_taxonomies['post_tag'] ) ) {
+			$wp_taxonomies['post_tag']->show_in_graphql     = true;
+			$wp_taxonomies['post_tag']->graphql_single_name = 'tag';
+			$wp_taxonomies['post_tag']->graphql_plural_name = 'tags';
+		}
 
-		add_filter(
-			'register_taxonomy_args',
-			function( $args, $taxonomy ) {
-
-				// Adds GraphQL support for categories.
-				if ( 'category' === $taxonomy ) {
-					$args['show_in_graphql']     = true;
-					$args['graphql_single_name'] = 'category';
-					$args['graphql_plural_name'] = 'categories';
-				}
-
-				// Adds GraphQL support for tags.
-				if ( 'post_tag' === $taxonomy ) {
-					$args['show_in_graphql']     = true;
-					$args['graphql_single_name'] = 'tag';
-					$args['graphql_plural_name'] = 'tags';
-				}
-
-				// Adds GraphQL support for post formats.
-				if ( 'post_format' === $taxonomy ) {
-					$args['show_in_graphql']     = true;
-					$args['graphql_single_name'] = 'postFormat';
-					$args['graphql_plural_name'] = 'postFormats';
-				}
-
-				return $args;
-
-			},
-			10,
-			2
-		);
-
+		// Adds GraphQL support for post formats.
+		if ( isset( $wp_taxonomies['post_format'] ) ) {
+			$wp_taxonomies['post_format']->show_in_graphql     = true;
+			$wp_taxonomies['post_format']->graphql_single_name = 'postFormat';
+			$wp_taxonomies['post_format']->graphql_plural_name = 'postFormats';
+		}
 	}
 
 	/**
@@ -492,8 +467,6 @@ final class WPGraphQL {
 	 * @since  0.0.4
 	 */
 	public static function get_allowed_post_types( $args = [] ) {
-
-		create_initial_post_types();
 
 		/**
 		 * Get all post_types
@@ -530,10 +503,11 @@ final class WPGraphQL {
 		 * to allow the post_types to be modified (for example if a certain post_type should
 		 * not be exposed to the GraphQL API)
 		 *
+		 * @since 0.0.2
+		 *
 		 * @param array $post_types Array of post types
 		 *
 		 * @return array
-		 * @since 0.0.2
 		 */
 		return apply_filters( 'graphql_post_entities_allowed_post_types', $post_types );
 
@@ -544,12 +518,10 @@ final class WPGraphQL {
 	 * are set to "show_in_graphql" but allows for external code (plugins/themes) to filter
 	 * the list of allowed_taxonomies to add/remove additional taxonomies
 	 *
-	 * @return array
 	 * @since  0.0.4
+	 * @return array
 	 */
 	public static function get_allowed_taxonomies() {
-
-		create_initial_taxonomies();
 
 		/**
 		 * Get all taxonomies
@@ -621,11 +593,11 @@ final class WPGraphQL {
 			/**
 			 * Generate & Filter the schema.
 			 *
-			 * @param array      $schema                 The executable Schema that GraphQL executes against
-			 * @param AppContext $app_context            Object The AppContext object containing all of the
-			 *                                           information about the context we know at this point
-			 *
 			 * @since 0.0.5
+			 *
+			 * @param array                 $schema      The executable Schema that GraphQL executes against
+			 * @param AppContext $app_context Object The AppContext object containing all of the
+			 *                                           information about the context we know at this point
 			 */
 			self::$schema = apply_filters( 'graphql_schema', $schema, self::get_app_context() );
 		}
@@ -677,11 +649,11 @@ final class WPGraphQL {
 			/**
 			 * Generate & Filter the schema.
 			 *
-			 * @param array      $type_registry            The TypeRegistry for the API
-			 * @param AppContext $app_context              Object The AppContext object containing all of the
-			 *                                             information about the context we know at this point
-			 *
 			 * @since 0.0.5
+			 *
+			 * @param array                 $type_registry The TypeRegistry for the API
+			 * @param AppContext $app_context   Object The AppContext object containing all of the
+			 *                                             information about the context we know at this point
 			 */
 			self::$type_registry = apply_filters( 'graphql_type_registry', $type_registry, self::get_app_context() );
 		}

--- a/tests/wpunit/NodeByUriTest.php
+++ b/tests/wpunit/NodeByUriTest.php
@@ -12,8 +12,6 @@ class NodeByUriTest extends \Codeception\TestCase\WPTestCase {
 
 	public function setUp(): void {
 
-		parent::setUp();
-		$this->set_permalink_structure( '/%year%/%monthnum%/%day%/%postname%/' );
 		WPGraphQL::clear_schema();
 
 		register_post_type('custom_type', [
@@ -21,16 +19,15 @@ class NodeByUriTest extends \Codeception\TestCase\WPTestCase {
 			'graphql_single_name' => 'CustomType',
 			'graphql_plural_name' => 'CustomTypes',
 			'public' => true,
-			'rewrite' => true,
 		]);
 
 		register_taxonomy( 'custom_tax', 'custom_type', [
 			'show_in_graphql' => true,
 			'graphql_single_name' => 'CustomTax',
 			'graphql_plural_name' => 'CustomTaxes',
-			'public' => true,
-			'rewrite' => true,
 		]);
+
+		$this->set_permalink_structure( '/%year%/%monthnum%/%day%/%postname%/' );
 
 		$this->user = $this->factory()->user->create([
 			'role' => 'administrator',
@@ -38,10 +35,7 @@ class NodeByUriTest extends \Codeception\TestCase\WPTestCase {
 
 		$this->tag = $this->factory()->term->create([
 			'taxonomy' => 'post_tag',
-			'name' => 'test'
 		]);
-
-//		wp_send_json( get_term_link( $this->tag ) );
 
 		$this->category = $this->factory()->term->create([
 			'taxonomy' => 'category',
@@ -68,11 +62,11 @@ class NodeByUriTest extends \Codeception\TestCase\WPTestCase {
 		$this->custom_type = $this->factory()->post->create( [
 			'post_type' => 'custom_type',
 			'post_status' => 'publish',
-			'post_title' => 'Test Custom Type',
+			'post_title' => 'Test Page',
 			'post_author' => $this->user
 		] );
 
-
+		parent::setUp();
 
 	}
 
@@ -90,6 +84,13 @@ class NodeByUriTest extends \Codeception\TestCase\WPTestCase {
 		wp_delete_term( $this->custom_taxonomy, 'custom_tax' );
 		wp_delete_user( $this->user );
 
+	}
+
+	public function set_permalink_structure( $structure = '' ) {
+		global $wp_rewrite;
+		$wp_rewrite->init();
+		$wp_rewrite->set_permalink_structure( $structure );
+		$wp_rewrite->flush_rules( true );
 	}
 
 	/**
@@ -257,12 +258,12 @@ class NodeByUriTest extends \Codeception\TestCase\WPTestCase {
 		}
 		';
 
-		codecept_debug( get_term_link( $this->category, 'category' ) );
+		codecept_debug( get_term_link( $this->category ) );
 
 		$actual = graphql([
 			'query' => $query,
 			'variables' => [
-				'uri' => get_term_link( $this->category, 'category' ),
+				'uri' => get_term_link( $this->category ),
 			],
 		]);
 
@@ -325,7 +326,7 @@ class NodeByUriTest extends \Codeception\TestCase\WPTestCase {
 		$actual = graphql([
 			'query' => $query,
 			'variables' => [
-				'uri' => get_term_link( $this->custom_taxonomy, 'custom_tax' ),
+				'uri' => get_term_link( $this->custom_taxonomy ),
 			],
 		]);
 

--- a/wp-graphql.php
+++ b/wp-graphql.php
@@ -6,7 +6,7 @@
  * Description: GraphQL API for WordPress
  * Author: WPGraphQL
  * Author URI: http://www.wpgraphql.com
- * Version: 1.3.1
+ * Version: 1.3.2
  * Text Domain: wp-graphql
  * Domain Path: /languages/
  * Requires at least: 5.0
@@ -18,7 +18,7 @@
  * @package  WPGraphQL
  * @category Core
  * @author   WPGraphQL
- * @version  1.3.1
+ * @version  1.3.2
  */
 
 // Exit if accessed directly.

